lib/organization_gem_dependencies/cli.rb

```diff
@@ -50,6 +53,29 @@ def run
           "#{gemfile.repository.name}/#{gemfile.path}"
         ))
       end
+
+      gemfiles(github, github_organization, GEMSPEC_SEARCH_TERM) do |gemspec|
```

Comment:
This block is nearly identical to the above save for the call to `process_` and the search term. Can we dry this up?